akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/HelloWorldPersistentEntityExample.java

```diff
@@ -29,7 +29,6 @@
 import akka.cluster.sharding.typed.javadsl.Entity;
 import akka.persistence.typed.javadsl.EventSourcedBehavior;
 import akka.serialization.jackson.CborSerializable;
-import akka.util.Timeout;
```

Comment:
This one is part of a code example, so it could be here on purpose even if unused, not sure